pkg/controller/registry/reconciler/reconciler.go

```diff
@@ -191,7 +191,7 @@ func Pod(source *operatorsv1alpha1.CatalogSource, name, opmImg, utilImage, img s
 						},
 					},
 					SecurityContext: &corev1.SecurityContext{
-						ReadOnlyRootFilesystem: ptr.To(false),
+						ReadOnlyRootFilesystem: ptr.To(true),
```

Comment:
As long as we support legacy access modes (and AFAIK we continue to do so), this needs to continue to be
```suggestion
						ReadOnlyRootFilesystem: ptr.To(false),
```